content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst

```diff
@@ -0,0 +1,112 @@
+============
+Stock report
+============
+
+Use the stock report in Odoo *Inventory* for a detailed list of all stored products, including those
+reserved, purchased and in transit, as well as those delivered to customers.
+
+.. note::
+   The reporting feature only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the stock report, go to :menuselection:`Inventory app --> Reporting --> Stock`.
+
+.. image:: stock/stock-report.png
+   :align: center
+   :alt: Show the stock report, accessible by going to Inventory > Reporting > Stock.
+
+Navigate the stock report
+=========================
+
+On the stock report, the left sidebar includes :guilabel:`Warehouses` to filter products by specific
+warehouses and :guilabel:`Category`, to show products within a selected product category.
+
+In the report itself, the columns represent:
+
+- :guilabel:`Product`: name of the product.
+- :guilabel:`Unit Cost`: average inventory valuation per unit, adjusted based on purchasing costs.
+- :guilabel:`Total Value`: Total inventory valuation of the product, calculated by multiplying unit
+  cost by on-hand quantity.
+
+.. seealso::
+   :doc:`../inventory_valuation/using_inventory_valuation`
+
+- :guilabel:`On Hand`: current quantity of products. Click the :icon:`fa-pencil`
+  :guilabel:`(pencil)` icon to :doc:`modify the on-hand quantity
+  <../inventory_management/count_products>`.
+- :guilabel:`Free to Use`: on-hand quantity that are not reserved for delivery or manufacturing
+  orders.
+- :guilabel:`Incoming`: items expected to arrive at the warehouse.
```

Comment:
Is this based on the number of approved/confirmed PO's? I might expand this so we know where this number is coming from. I assume it's fro the PO- but is it nly ocunting confirmed, any on order, quuotes requested, etc?